CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Changed
 - Changed deprecated `spaceless` into `apply spaceless` in Twig template
+### Added
```

Comment:
Wat mij betreft mag je hier gewoon een `6.2.0` versietag aan hangen, het is immers een nieuwe feature. Anders heb je kans dat het met een getagde bugfix meekomt, wat semantisch niet helemaal klopt :thinking:

Reply:
Meestal klus ik die er na approve of na merge er in. Het kan zo maar zo zijn dat zo'n PR een week open staat en iemand anders er nog een versie tussendoor fietst, maar bij deze... `v6.2.0`